subprojects/build-init/src/integTest/groovy/org/gradle/buildinit/plugins/AbstractInitIntegrationSpec.groovy

```diff
@@ -58,12 +59,38 @@ abstract class AbstractInitIntegrationSpec extends AbstractIntegrationSpec {
         assertNoDefinedBuild(targetDir)
     }
 
+    void assertTestsDidNotRun(String className) {
+        def result = new DefaultTestExecutionResult(subprojectDir)
```

Comment:
It was a little more complicated, because of how some of the `TestExecutionResult` subclasses made assumptions that a least one test would have been run, so I just did it this way to get something working quickly.  I've replaced with the proper methods for each implementation.